CriticalMass/MapInfoViewController.swift

```diff
@@ -117,6 +140,20 @@ class MapInfoViewController: UIViewController, IBConstructable {
             animator.startAnimation()
         }
     }
+
+    @objc private func didReceiveLocationsUpdate(notification: Notification) {
+        guard
+            let result = notification.object as? ApiResponseResult,
+            result.isError().0 != showServerError
+        else {
+            return
+        }
+        let errorMessage = (result.isError().1 as! NetworkError).errorDescription ?? L10n.Map.Layer.Info.errorMessage
+        errorView.setErrorLabelMessage(errorMessage)
+        showServerError = result.isError().0
+        UIAccessibility.post(notification: .layoutChanged, argument: nil)
```

Comment:
I think this should rather be an announcement with the error message as body. But I'm not 100% sure. 

```suggestion
        UIAccessibility.post(notification: .announcement, argument: errorMessage)
```

Reply:
we can leave it like that, because all other Accessibility posts in this view controller use `.layoutChanged` as well.